src/main/java/com/iota/iri/storage/PersistenceProvider.java

```diff
@@ -1,10 +1,10 @@
 package com.iota.iri.storage;
 
-import com.iota.iri.model.*;
 import com.iota.iri.utils.Pair;
+import org.rocksdb.RocksDBException;
```

Comment:
same, the `PersistenceProvider` shouldn't be dependant on a specific implementation - even if it's a silly thing like an Exception.

Reply:
:+1: